defaults/main.yml

```diff
@@ -24,7 +24,7 @@ besu_profile_file: "/etc/profile.d/teku-path.sh"
 teku_log_level: INFO
 teku_log_color_enabled: False
 teku_log_destination: 'DEFAULT_BOTH'
-teku_log_filename: 'teku.log'
+teku_log_filename: "{{ teku_log_dir }}/teku.log"
```

Comment:
Does this work properly when using a remote validator? They should wind up with two separate log files (one `teku.log` and one `teku-validator.log` usually)

Reply:
Let me check

Reply:
Its no worse than it was but may be something we should follow up with.

Reply:
Looks like both configure to write to same folder at least when not using the log4j. I will update this so that they write to two separate logs.

Reply:
I will change the log file in separate MR and merge this now. Currently this change is critical for the correct deployment.